### PR TITLE
Temperature Widget HTML

### LINKED ISSUE #52: Create HTML for temperature widget

### PR DESCRIPTION
**Pull request by robertyac**
Initial **HTML only** design for temp widget:

Screenshot 2024-03-06 at 6 19 44 PM
Screenshot 2024-03-06 at 6 16 31 PM

potentially closes #52

### CODE REVIEW

File: Site/bc_weather_station_dashboard/website/templates/home.html [hunk 1]
```diff
@@ -92,15 +91,42 @@ <h2>
...
+                        <div class="card-body text-center p-4">
+                            <!-- Temp icon and title -->
+                            <h5 class="card-title mb-2 d-flex align-items-center justify-content-center">
+                                <span class="mr-2">
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="white"
+                                        class="bi bi-thermometer" viewBox="0 0 16 16">
+                                        <path d="M8 14a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3" />
+                                        <path
+                                            d="M8 0a2.5 2.5 0 0 0-2.5 2.5v7.55a3.5 3.5 0 1 0 5 0V2.5A2.5 2.5 0 0 0 8 0M6.5 2.5a1.5 1.5 0 1 1 3 0v7.987l.167.15a2.5 2.5 0 1 1-3.333 0l.166-.15z" />
+                                    </svg>
```
Comment (cmcahoon01) [inline Site/bc_weather_station_dashboard/website/templates/home.html]: This should be an svg in the static folder, not an inline tag.

**Review by cmcahoon01 — CHANGES_REQUESTED** [review]
Other than the one comment lgtm

File: Site/bc_weather_station_dashboard/website/templates/home.html (hunk 1 above)
Comment (robertyac) [reply Site/bc_weather_station_dashboard/website/templates/home.html:106]: what file should it go in?

Comment (robertyac) [reply Site/bc_weather_station_dashboard/website/templates/home.html:106]: nvm

Comment (cmcahoon01) [reply Site/bc_weather_station_dashboard/website/templates/home.html]: look how the calendar svg is saved. It should be its own file
image

Comment (robertyac) [reply Site/bc_weather_station_dashboard/website/templates/home.html]: yeah I made a folder for images in static and put it there 👍🏻

**Review by cmcahoon01 — APPROVED** [review]

**Review by abijeet-dhillon — APPROVED** [review]
LGTM!

**Pull request merged**